coriolis_openstack_utils/conf.py

```diff
@@ -107,6 +107,11 @@
     help="String format for flavor names on the destination. "
          "Must contain the format string '%(original)s'. "
          "Example: %(original)s-Migrated")
+NEW_KEYPAIR_NAME_OPT = conf.StrOpt(
+    "new_keypair_name_format", required=True,
+    help="String format for keypair names on the destination. "
+         "Must contain the format string '%(original)s'. "
```

Comment:
`Must` is a strong word (when formatting dicts nothing happens if a key in the dict isn't in the format string, so people can just set this to `oregon_trail` if they want all their migrated keypairs to be named that way...)